dat/BIOS (Non-Merged).dat

```diff
@@ -1,62 +1,52 @@
 clrmamepro (
-	name "libretro - Bios"
-	description "Bios images for libretro cores."
-	category "libretro"
-	version "0.0.1"
-	date "2016-06-05"
-	author "libretro Team"
+	name "Libretro - BIOS (Non-Merged)"
+	version "20161204"
 	homepage "http://github.com/libretro/libretro-database"
 )
 
 game (
-	name "The 3DO Company - 3DO"
-	description "The 3DO Company - 3DO"
-	rom ( name goldstar.bin size 1048576 crc b6f5028b md5 8639fd5e549bd6238cfee79e3e749114 sha1 c4a2e5336f77fb5f743de1eea2cda43675ee2de7 )
-	rom ( name panafz1.bin size 1048576 crc c8c8ff89 md5 f47264dd47fe30f73ab3c010015c155b sha1 34bf189111295f74d7b7dfc1f304d98b8d36325a )
-	rom ( name panafz10.bin size 1048576 crc 58242cee md5 51f2f43ae2f3508a14d9f56597e2d3ce sha1 3c912300775d1ad730dc35757e279c274c0acaad )
-	rom ( name sanyotry.bin size 1048576 crc d5cbc509 md5 35fa1a1ebaaeea286dc5cd15487c13ea sha1 b01c53da256dde43ffec4ad3fc3adfa8d635e943 )
```

Comment:
Found the references to 3DO in MAME:
https://github.com/libretro/mame2014-libretro/blob/2e3943dd6e9bf6e4161704179d7af2fbbd313e92/src/mame/drivers/3do.c#L194-L204